docs/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :bug:`8777` Fix database cursor reuse causing zksync lite transactions to be skipped.
```

Comment:
The changelog text is user facing. Don't mention details like this. it's greek to them.

"ZKSync lite transactions should no longer be skipped under special circumstances"